src/python/gudhi/persistence_graphical_tools.py

```diff
@@ -364,61 +386,46 @@ def plot_persistence_density(
 
     :Requires: `SciPy <installation.html#scipy>`_
 
-    :param persistence: Persistence intervals values list. 
-                        Can be grouped by dimension or not.
-    :type persistence: an array of (dimension, array of (birth, death)) 
-                        or an array of (birth, death).
+    :param persistence: Persistence intervals values list. Can be grouped by dimension or not.
+    :type persistence: an array of (dimension, (birth, death)) or an array of (birth, death)
     :param persistence_file: A `persistence diagram <fileformats.html#persistence-diagram>`_
         file style name (reset persistence if both are set).
     :type persistence_file: string
-    :param nbins: Evaluate a gaussian kde on a regular grid of nbins x
-        nbins over data extents (default is 300)
-    :type nbins: int.
-    :param bw_method: The method used to calculate the estimator
-        bandwidth. This can be 'scott', 'silverman', a scalar constant
-        or a callable. If a scalar, this will be used directly as
-        kde.factor. If a callable, it should take a gaussian_kde
-        instance as only parameter and return a scalar. If None
-        (default), 'scott' is used. See
+    :param nbins: Evaluate a gaussian kde on a regular grid of nbins x nbins over data extents (default is 300)
+    :type nbins: int
+    :param bw_method: The method used to calculate the estimator bandwidth. This can be 'scott', 'silverman', a scalar
+        constant or a callable. If a scalar, this will be used directly as kde.factor. If a callable, it should take a
+        gaussian_kde instance as only parameter and return a scalar. If None (default), 'scott' is used. See
         `scipy.stats.gaussian_kde documentation
         <http://scipy.github.io/devdocs/generated/scipy.stats.gaussian_kde.html>`_
         for more details.
-    :type bw_method: str, scalar or callable, optional.
-    :param max_intervals: maximal number of points used in the density
-        estimation.
-        Selected intervals are those with the longest life time. Set it
-        to 0 to see all. Default value is 1000.
-    :type max_intervals: int.
-    :param dimension: the dimension to be selected in the intervals
-        (default is None to mix all dimensions).
-    :type dimension: int.
-    :param cmap: A matplotlib colormap (default is
-        matplotlib.pyplot.cm.hot_r).
-    :type cmap: cf. matplotlib colormap.
-    :param legend: Display the color bar values (default is False).
-    :type legend: boolean.
-    :param axes: A matplotlib-like subplot axes. If None, the plot is drawn on
-        a new set of axes.
+    :type bw_method: str, scalar or callable, optional
+    :param max_intervals: maximal number of points used in the density estimation. Selected intervals are those with
+        the longest life time. Set it to 0 to see all. Default value is 1000.
```

Comment:
Unrelated to this PR: we will have to do something about it at some point, the whole point of plotting a density is for the case where there are too many points. If we ignore most of the points...

Reply:
Yes you are right. Thank you for the opening of #801